examples/test_suite_1/test_suite_1.py

```diff
@@ -162,6 +162,7 @@ def test_run(self):
         logging.info(f"------------resume auxiliaries run--------------")
         aux3.resume()
         aux2.resume()
+        print(aux4.get_message("BHU", ["KeyExchange_Master_BHU", "BHU_NetworkStatus"]))
```

Comment:
Remove this line. 
In this repo always generic examples. Nothing from your workplace

Reply:
Please leave the dummy example untouched in this context.. This example shall always run without external HW.